neurodocker/cli/minify/tests/test_minify.py

```diff
@@ -8,11 +9,28 @@
 docker = pytest.importorskip("docker", reason="docker-py not found")
 
 
-@pytest.mark.skip(reason="ptrace no longer supported under docker")
+def _arm_on_mac():
+    """Return True if on an ARM processor (M1/M2) in macos operating system."""
+    is_mac = platform.system().lower() == "darwin"
+    is_arm = platform.processor().lower() == "arm"
+    return is_mac and is_arm
+
+
+skip_arm_on_mac = pytest.mark.skipif(
+    _arm_on_mac(), reason="minification does not work on M1/M2 macs"
+)
+
+
+@skip_arm_on_mac
 def test_minify():
     client = docker.from_env()
-    container = client.containers.run("python:3.9-slim", detach=True, tty=True,
-                                      platform="Linux/amd64", privileged=True)
+    container = client.containers.run(
+        "python:3.9-slim",
+        detach=True,
+        tty=True,
+        platform="Linux/amd64",
+        privileged=True,
```

Comment:
is this line needed for ptrace to work? if so it should be added to the docs as well.